runner/execer/execer.go

```diff
@@ -30,8 +30,7 @@ type Execer interface {
 type Process interface {
```

Comment:
Added some very basic comments. To elaborate slightly, every combination of calls returns process.FAILED except a solitary call to Wait() when the process exits cleanly.